tests/builder/interpolation_test.py

```diff
@@ -1,535 +0,0 @@
-        # parameters for the weight axis. For the width axis, glyphsLib
-        # should use 100 as default value (just like Glyphs.app does).
-        familyName = "DesignspaceTest TwoAxes"
-        masters = [
-            makeMaster("Regular", weight=90),
-            makeMaster("Black", weight=190),
-            makeMaster("Thin", weight=26),
-            makeMaster("ExtraCond", weight=90, width=70),
-            makeMaster("ExtraCond Black", weight=190, width=70),
-            makeMaster("ExtraCond Thin", weight=26, width=70),
-        ]
-        instances = [
-            makeInstance("Thin", weight=("Thin", 100, 26)),
-            makeInstance("Regular", weight=("Regular", 400, 90)),
-            makeInstance("Semibold", weight=("SemiBold", 600, 128)),
-            makeInstance("Black", weight=("Black", 900, 190)),
-            makeInstance("ExtraCondensed Thin",
-                         weight=("Thin", 100, 26),
-                         width=("Extra Condensed", 2, 70)),
-            makeInstance("ExtraCondensed",
-                         weight=("Regular", 400, 90),
-                         width=("Extra Condensed", 2, 70)),
-            makeInstance("ExtraCondensed Black",
-                         weight=("Black", 900, 190),
-                         width=("Extra Condensed", 2, 70)),
-        ]
-        font = makeFont(masters, instances, familyName)
-        doc = to_designspace(font, instance_dir='out')
-        self.expect_designspace(doc, "DesignspaceTestTwoAxes.designspace")
-        self.expect_designspace_roundtrip(doc)
-
-    def test_variationFontOrigin(self):
-        # Glyphs 2.4.1 introduced a custom parameter “Variation Font Origin”
-        # to specify which master should be considered the origin.
-        # https://glyphsapp.com/blog/glyphs-2-4-1-released
-        masters = [
-            makeMaster("Thin", weight=26),
-            makeMaster("Regular", weight=100),
-            makeMaster("Medium", weight=111),
-            makeMaster("Black", weight=190),
-        ]
-        instances = [
-            makeInstance("Black", weight=("Black", 900, 190)),
-            makeInstance("Medium", weight=("Medium", 444, 111)),
-            makeInstance("Regular", weight=("Regular", 400, 100)),
-            makeInstance("Thin", weight=("Thin", 100, 26)),
-        ]
-        font = makeFont(masters, instances, "Family")
-        font.customParameters["Variation Font Origin"] = "Medium"
-        designspace = to_designspace(font, instance_dir='out')
-        path = self.write_to_tmp_path(designspace, 'varfontorig.designspace')
-        doc = etree.parse(path)
-        medium = doc.find('sources/source[@stylename="Medium"]')
-        self.assertEqual(medium.find("lib").attrib["copy"], "1")
-        weightAxis = doc.find('axes/axis[@tag="wght"]')
-        self.assertEqual(weightAxis.attrib["default"], "444")
-
-        self.expect_designspace_roundtrip(designspace)
-
-    def test_designspace_name(self):
```

Comment:
why did you remove the whole `interpolation_test.py` module? there are many tests that still apply to the current code. you can rename the test module (or move the relevant test elsewhere) if the lib module that it was originally meant to be testing is no longer present

Reply:
Oops, I was overeager.

I noticed that fontmake relies on build_instances, though :(

Reply:
> fontmake relies on build_instances

no, where? You mean "apply_instance_data" perhaps?

Reply:
Err, yeah, that one.